hugo/assets/scripts/domainmodel.tsx

```diff
@@ -0,0 +1,116 @@
+import { MonacoEditorReactComp, monaco, addMonacoStyles } from '@typefox/monaco-editor-react/bundle';
+import { buildWorkerDefinition } from 'monaco-editor-workers';
+import React from 'react';
+import { createRoot } from 'react-dom/client';
+
+buildWorkerDefinition(
+  "../../libs/monaco-editor-workers/workers",
+  new URL("", window.location.href).href,
+  false
+);
+addMonacoStyles('monaco-editor-styles');
+
+const syntaxHighlighting = {
+  keywords: [
+    'datatype', 'entity', 'extends', 'many', 'package'
+  ],
+  operators: [
+    ':', '.'
+  ],
+  symbols: /:|\.|\{|\}/,
+
+  tokenizer: {
+    initial: [
+      { regex: /[_a-zA-Z][\w_]*/, action: { cases: { '@keywords': { "token": "keyword" }, '@default': { "token": "ID" } } } },
+      { regex: /[0-9]+/, action: { "token": "number" } },
+      { regex: /"[^"]*"|'[^']*'/, action: { "token": "string" } },
+      { include: '@whitespace' },
+      { regex: /@symbols/, action: { cases: { '@operators': { "token": "operator" }, '@default': { "token": "" } } } },
+    ],
+    whitespace: [
+      { regex: /\s+/, action: { "token": "white" } },
+      { regex: /\/\*/, action: { "token": "comment", "next": "@comment" } },
+      { regex: /\/\/[^\n\r]*/, action: { "token": "comment" } },
+    ],
+    comment: [
+      { regex: /[^\/\*]+/, action: { "token": "comment" } },
+      { regex: /\*\//, action: { "token": "comment", "next": "@pop" } },
+      { regex: /[\/\*]/, action: { "token": "comment" } },
+    ],
+  }
+} as monaco.languages.IMonarchLanguage;
+
+
+class App extends React.Component<{}, { hasError }> {
+  monacoEditor: React.RefObject<MonacoEditorReactComp>;
+  constructor(props) {
+    super(props);
+    this.onTextChanged = this.onTextChanged.bind(this);
+    this.monacoEditor = React.createRef();
+    this.state = {
+      hasError: false
+    };
+  }
+
+  onTextChanged(text: string, isDirty: boolean) {
```

Comment:
Same here, must have been a vestige from before. Can drop this function, its binding above, and its usage below.